README.md

```diff
@@ -647,3 +662,16 @@ For example, if a=[6, 2, 3, 4, 12], then you need to use two colors: 6,3,12 can
 **Input:** First row contains one integer n (1<=n<=100), where n is the lenght of the sequence. Second row contains n integers a1,a2,...,an (1<=ai<=100). Numbers in the sequence can be repeated.
 
 **Output:** Print the minimal number of colors needed to color the elements correctly.
+
+### Insert Delete GetRandom O(1) - [Java](java/src/main/java/ru/nk/training/RandomizedSet.java)
+
+[LeetCode - Insert Delete GetRandom O(1)](https://leetcode.com/problems/insert-delete-getrandom-o1/)
+
+Implement the `RandomizedSet` class:
+
+- `RandomizedSet()` Initializes the `RandomizedSet` object.
+- `bool insert(int val)` Inserts an item val into the set if not present. Returns true if the item was not present, false otherwise.
```

Comment:
```suggestion
- `bool insert(int val)` Inserts an item `val` into the set if not present. Returns `true` if the item was not present, `false` otherwise.
```